docs/docs/deployment.md

```diff
@@ -193,6 +206,53 @@ cfg := &tls.Config{
 </TabItem>
 </Tabs>
 
+### Using GPUs
+
+If the cluster is [configured for GPU usage](./getting-started/bare-metal.md#preparing-a-cluster-for-gpu-usage), Pods can use GPU devices if needed.
+
+To do so, a CDI annotation needs to be added, specifying to use the `pgpu` (passthrough GPU) mode. The `0` corresponds to the PCI device index.
+* For nodes with a single GPU, this value is always `0`.
+* For nodes with multiple GPUs, the value needs to correspond to the device's order as enumerated on the PCI bus. You can identify this order by inspecting the `/var/run/cdi/nvidia.com-pgpu.yaml` file on the specific node.
+
+This process ensures the correct GPU is allocated to the workload.
+
+As the footprint of a GPU-enabled pod-VM is larger than one of a non-GPU one, the memory of the pod-VM can be adjusted by using the `io.katacontainers.config.hypervisor.default_memory` annotation, which receives the memory the
```

Comment:
afaik this annotation should only cover the guest os & kata components, not the gpu workload itself (workload memory is adjusted using resource limits). So I don't get why we need 8/16GB here.

Reply:
This is the memory alloted to the VM. I don't know what you mean exactly here. Trying to start a GPU VM with e.g. 4GB of memory fails while starting up the VM.